examples/3d-heatmap/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-const {resolve} = require('path');
```

Comment:
Might be worth adding a comment stating that we do this to support Node 4 so that some helpful maintainer doesn't revert in a future commit. Probably worth doing in every file.

PS - These days my thinking is that almost every line in webpack configs should have a comment it is not an easy file to read.